CMakeLists.txt

```diff
@@ -92,6 +92,12 @@ function(configure_cesium_library targetName)
         CXX_EXTENSIONS NO
     )
 
+    target_compile_definitions( ${targetName} PRIVATE
```

Comment:
I think we want `PUBLIC` rather than `PRIVATE`, so that other libraries that depend on cesium-native via cmake will use the same definitions.

Reply:
Do you know how to make these values propagate? Doesn't look like it's happening automagically

Reply:
I was under the impression that they'll propagate automatically if they're declared PUBLIC, but I've never really tested it.

Reply:
I think the problem is I'm using ExternalProject_Add for cesium-native so it doesn't have a real target to work with. But I changed the PR to be `PUBLIC` so that it works with `add_subdirectory` or similar